### DISCUSSION TITLE
Carlinkit 3.0 not broadcasting BT or WiFi? Only USB working...

### DISCUSSION THREAD

**Discussion by pavl0s** [ludwig-v/wireless-carplay-dongle-reverse-engineering#125] [Q&A]
I have two Carlinkit 3.0 devices (**CPC200-U2W Plus** device label), first one works perfectly, the second one is having this issue. For completeness I have tried them in the same cars and multiple phones, I am familiar with how to connect them, so 99.9% sure it is a device issue.

Firstly, I'm trying to determine if the problem is a hardware fault or simply a software/firmware issue. **The device does not broadcast any wifi network/SSID and when pressing the "Search" button to initiate a bluetooth pairing connection it does not do anything at all.** The only buttons that work are the settings button and back button to exit.

The device powers on OK and red light appears, it boots in to the red autokit screen UI on the car but that is all. I can flash firmware over USB without any issue (red/green lights appear). I have already tried stock firmware versions 2020.12.25.1732 [0110de0170fbCMY], 2021.03.06.1355 [0670cd023105CMY], 2021.07.05.2319 [0670d10470fcCMY], 2021.08.24.1817 [06c0e30510f1CMY], which I confirmed by reconnecting to the car and booting each time after an update and autokit UI shows those respective software versions on the settings screen. 

Two things I have noticed when comparing with the working dongle.
1. There is no device name that appears on the autokit UI screen, i.e. rather than "CARMANU-XY12" all that appears is ".."
2. In the settings screen the CMID is blank - what is this?

Also worth noting when I plug in a USB lightning cable to the other end of the dongle, Carplay will work through the dongle without issue. It just appears that the dongle wireless features are not working, and I wonder if this is because the device is not assigning itself a name therefore causing the BT/WIFI network initiation processes to fail?

So can anyone help with...
1. Given I can't connect to a wifi ID that doesn't exist, is there any other way to troubleshoot just with the USB connection?
2. Is there any way to initiate a full device reset over USB normally shown in the web UI 192.168.50.2 which per above I can't connect to?
3. Will any of the custom firmware help fix these issues?
4. Is this is a hardware/software issue?

Thanks

**Comment by magflip** [comment]
Hello. I have the same problem. Have you found any solution?
